contentcuration/contentcuration/frontend/channelList/views/Channel/ChannelModal.vue

```diff
@@ -227,9 +243,16 @@
 
 <style lang="less" scoped>
 
-  .title {
-    margin: 25px 0 10px;
-    font-weight: bold;
+  /deep/ fieldset {
+    border: 0;
+
+    .title {
+      font-size: 18px !important;
+    }
+  }
```

Comment:
None of this is a blocker, but just noting for future reference some things I often flag in Kolibri code:

* I usually prefer to create a new class (or just use an inline `style="border: 0"`) instead of referencing a tag name, in case someone comes along and adds another tag later
* I usually consider `/deep/` to be a code-smell because it breaks `scoped` style boundaries
* I usually advise to avoid nesting CSS because it increases specificity and coupling with DOM structure. See similar comment here: https://github.com/learningequality/studio/pull/1734#discussion_r370420670
* `!important` is generally [to be avoided](https://uxengineer.com/css-specificity-avoid-important-css/)

Reply:
I will make these changes, but have some thoughts:

> I usually prefer to create a new class 

Yeah, I would too, but since I also want to deeply select the `fieldset` in the `ContentDefaults.vue`, I see using a class for that as having higher entropy for a design bug should someone change the class here or both, but perhaps not all uses of `ContentDefaults`. Additionally, I think this also communicates a better question: do we want a global style to remove the border on `fieldset`, which IIRC came from Vuetify?

> I usually consider /deep/ to be a code-smell

I tend to avoid adding styling to shared components to avoid the need of selector specificity in overriding them in a parent's case.

> I usually advise to avoid nesting CSS

In my frontend experience, I've gone both ways. I feel like much of the arguments are pretty theoretical, and IMO only the increase in compiled CSS size really carries the most weight. I've seen [BEM](http://getbem.com/) carried out and no real benefits realized.  Now, I'd agree that BEM with scoped styling isn't really necessary, but if we feel like this is important enough, choosing a standard like that would be helpful.

> !important is generally to be avoided

Agreed. In the Vuetify styling, there wasn't a size that matched the design, and unfortunately, I could not override the font without it. The alternate would be using a different class name that doesn't inherit the `.title` styling and creates the same effect.